### PR TITLE
Galaxy NG packaging

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/pulpcore/python-galaxy-ng/python-galaxy-ng.spec [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+%global full_version %{version}a10
+
+Name:           python-%{pypi_name}
+Version:        4.2.0
+Release:        0.1.a10%{?dist}
+Summary:        galaxy-ng plugin for the Pulp Project
+
+License:        GPLv2+
+URL:            http://example.com/
```
Comment (evgeni) [inline packages/pulpcore/python-galaxy-ng/python-galaxy-ng.spec:12]: Yes, that's in setup.py ;)

**Comment by evgeni** [comment]
Hmm, need to patch out certifi (or upload a patched one, whatever)

**Comment by evgeni** [comment]
```
DEBUG util.py:600:    Status code: 404 for http://mirror.centos.org/centos/8/BaseOS/x86_64/os/Packages/libnghttp2-1.33.0-3.el8_2.1.x86_64.rpm (IP: 64.150.179.24)
```

*sigh²*

@pcreech do you see why this RPM isn't found?

**Comment by pcreech** [comment]
[test rpm]

**Pull request merged**